apps/client/src/pages/auth/_components/social-auth.tsx

```diff
@@ -32,6 +32,20 @@ export const SocialAuth = () => {
           </a>
         </Button>
       )}
+
+      {providers.includes('oidc') && (
+
+        <Button
+          asChild
+          size="lg"
+          className="w-full !bg-[#4285F4] !text-white hover:!bg-[#4285F4]/80"
+        >
+          <a href="/api/auth/oidc">
+            <Lock className="mr-3 size-4" />
+            Generic OIDC
```

Comment:
The ability to name the login would be nice. Aka. if you have an Authentik instance you called "SSO", you can have the button say "Sign-in with SSO". Maybe this can be an environment variable?